plotly/offline/offline.py

```diff
@@ -327,9 +330,9 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
     if not ipython:
         raise ImportError('`iplot` can only run inside an IPython Notebook.')
 
-    config = {}
-    config['showLink'] = show_link
-    config['linkText'] = link_text
+    config = dict(config) if config else {}
+    config.setdefault('showLink', show_link)
```

Comment:
Why not just write `config['showLink] = show_link`? It's clearer to me.

Same goes with `config.setdefault('linkText', link_text)`